uqcsbot/text.py

```diff
@@ -80,6 +80,14 @@ async def mock(self, ctx: commands.Context, *text: str):
         """
         await ctx.send("".join(choice((c.upper(), c.lower())) for c in " ".join(text)))
 
+    @commands.command()
+    async def scare(self, ctx: commands.Context, *text: str):
+        """
+        "adds" "scary" "quotes" "around" "the" "provided" "text"
+        """
+
+        await ctx.send(" ".join(f'"{w}"' for w in text.split(' ')))
```

Comment:
```suggestion
        await ctx.send(" ".join(f'"{w}"' for w in text))
```

Reply:
won't this just put quotes around every character?

Reply:
nvm

Reply:
can we make !extrascare that does that pls